functional/FunctionalTest.java

```diff
@@ -2779,6 +2780,74 @@ public static void removeAllBucketNotification_test1() throws Exception {
     }
   }
 
+  /**
+   * Test: listenBucketNotification(String bucketName).
+   */
+  public static void listenBucketNotification_test1() throws Exception {
+    if (!mintEnv) {
+      System.out.println("Test: listenBucketNotification(String bucketName)");
+    }
+
+    long startTime = System.currentTimeMillis();
+    try {
+      String bucketName = getRandomName();
+      client.makeBucket(bucketName, region);
+
+      class TestBucketListener implements BucketEventListener {
+        int eventsReceived = 0;
+        NotificationEvent firstEvent = null;
+
+        @Override
+        public void updateEvent(NotificationInfo info) {
+          if (info.records != null && info.records.length > 0) {
+            firstEvent = info.records[0];
+          }
+          this.eventsReceived++;
+        }
+      }
+
+      TestBucketListener bl = new TestBucketListener();
+
+      new Thread(new Runnable() {
+        public void run() {
+          String[] events = {"s3:ObjectCreated:*", "s3:ObjectAccessed:*"};
+          try {
+            client.listenBucketNotification(bucketName, "prefix", "suffix", events, bl);
+          } catch (Exception e) {
+            System.out.println(e);
+          }
+        }}).start();
+
+      String file1kb = createFile(1024);
```

Comment:
./gradlew build
Indentation issue at line 2821 `String file1kb = createFile(1024);` 
> Task :api:checkstyleMain FAILED
[ant:checkstyle] /home/ashish/go/src/github.com/minio/minio-java/functional/FunctionalTest.java:2822: 'member def type' have incorrect indentation level 7, expected level should be 6.

Reply:
> ./gradlew build

Fixed.